src/tech_node_data/asap7data.txt

```diff
@@ -0,0 +1,438 @@
+cell (AND2x2_ASAP7_75t_L) 
```

Comment:
We don't need this file once we have the other one. But I would keep it somewhere locally so that we can access it later as an archive if need be. Or you can upload to gdrive and share it. 

It'll just add clutter in the repo imo.